.github/workflows/TriggerReferenceDocsCI.yml

```diff
@@ -1,16 +1,17 @@
 name: Trigger Microsoft Learn Docs Reference CI
 
 on:
-  push:
-    branches:
-      - main
+  release:
+    types:
+      - released

```

Comment:
Ref: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#release